js/lightbeam.js

```diff
@@ -20,6 +24,82 @@ const lightbeam = {
     });
   },
 
+  // Called from init() (isFirstParty = undefined)
+  // and redraw() (isFirstParty = true or false).
+  async updateVars(isFirstParty) {
+
+    // initialize dynamic vars from storage
+    if (!this.dataGatheredSince) {
+      const { dateStr, fullDateTime } = await this.getDataGatheredSince();
+      this.dataGatheredSince = dateStr;
+      const dataGatheredSince = document.getElementById('data-gathered-since');
+      dataGatheredSince.textContent = this.dataGatheredSince;
```

Comment:
@biancadanforth same variable name `dataGatheredSince` is a bit confusing. can we rename the second one to `dataGatheredSinceElement` or maybe `this.dataGatheredSince` to `this.dateStr`?